nmtwizard/preprocess/sampler.py

```diff
@@ -260,7 +260,7 @@ def _select_lines(f):
 
     # TODO V2 : multiple sources and targets
     src_suffix=config["source"]
-    tgt_suffix=config["target"]
+    tgt_suffix=config.get("target", None)
```

Comment:
This part worries me a little because it will allow missing/null target suffix for standard preprocess as well.
Preprocess will proceed and then generate some obscure errors later.

Not sure about the best way to avoid it.

Reply:
I added an assert to check the config, but not sure it's the best way.

Reply:
Maybe add an explicit error message as second argument to the assert ?